.ci/dockerfiles/Dockerfile.cuda-manylinux

```
@@ -0,0 +1,44 @@
+#     -f .ci/dockerfiles/Dockerfile.cuda-manylinux -t $REPO:$TAG-x86_64 . && docker push $REPO:$TAG-x86_64
+#   # aarch64 host:
+#   docker build --platform linux/arm64 --build-arg ARCH=aarch64 \
+#     -f .ci/dockerfiles/Dockerfile.cuda-manylinux -t $REPO:$TAG-aarch64 . && docker push $REPO:$TAG-aarch64
+#   docker manifest create $REPO:$TAG $REPO:$TAG-x86_64 $REPO:$TAG-aarch64 && docker manifest push $REPO:$TAG
+#   # then set the cu12 nightly cron's BASE_TAG to $TAG
+
+ARG ARCH=x86_64
+ARG CUDA_VERSION=12.9.1
+
+FROM nvcr.io/nvidia/cuda:${CUDA_VERSION}-devel-ubi8 AS cuda
+FROM quay.io/pypa/manylinux_2_28_${ARCH}
```

Hadolint (2.14.0)

[warning] Always tag the version of an image explicitly

(DL3006)